contents.json

```diff
@@ -4852,6 +4852,12 @@
       "category": "animation",
       "description": "Poi makes you use card UI like tinder UI .You can use it like tableview method.",
       "homepage": "https://github.com/HideakiTouhara/Poi"
+    }, {
+	    "title": "DNWebSocket",
+	    "category": "socket",
+	    "description": "Object-Oriented, Swift-style WebSocket Library (RFC 6455) for Swift-compatible Platforms.",
```

Comment:
I'm not sure about using so many `Swift` in the description. What do you think guys @dkhamsing, @matteocrippa?

Reply:
I can replace description, it's not a problem :)